include/SDL3/SDL_mouse.h

```diff
@@ -521,7 +521,7 @@ extern DECLSPEC SDL_bool SDLCALL SDL_CursorVisible(void);
  *
  * \since This macro is available since SDL 3.0.0.
  */
-#define SDL_BUTTON(X)       (1 << ((X)-1))
+#define SDL_BUTTON(X)       (1u << ((X)-1))
 
 #define SDL_BUTTON_LEFT     1
 #define SDL_BUTTON_MIDDLE   2
```

Comment:
Shall the `SDL_BUTTON_...` macros also have a `u` suffix?
In `SDL_MouseButtonEvent` the `button` member also is `Uint8`.

Reply:
Out of scope for this PR.